dir.traversal.targets

```diff
@@ -60,6 +60,11 @@
     <!-- To Serialize we use msbuild's batching functionality '%' to force it to batch all similar projects with the same identity
          however since the project names are unique it will essentially force each to run in its own batch -->
 
+    <!-- Coverage builds require serial execution. -->
+    <PropertyGroup>
+      <SerializeProjects Condition="'$(Coverage)' == 'true'">true</SerializeProjects>
```

Comment:
I saw this, too, but it isn't affecting the traversal build of tests.builds, for some reason. Perhaps we refactored something and tests.targets isn't imported at the right time in that project anymore?